tchannel/tornado/dispatch.py

```diff
@@ -160,27 +165,24 @@ def check_health(request, response, proxy):
             A function that gets called with ``Request``, ``Response``, and
             the ``proxy``.
 
-        :param broker:
-            Broker injects customized serializer and deserializer into
-            request/response object.
+        :param req_serializer:
+            Arg scheme serializer of this endpoint. It should be
+            ``RawSerializer``, ``JsonSerializer``, and ``ThriftSerializer``.
```

Comment:
Well, I mean it doesn't have to be one of those three. As long as it satisfies the `Serializer` interface, it's a valid argument here.

Actually, can you add a `Serializer` class that just defines the interface (`(de)serialize_{headers,body}`)? You can make the Raw, JSON, and Thrift serializers inherit it.